toolchains/features/embedded/impl/gcc.bzl

```diff
@@ -90,7 +90,8 @@ _SYS_SPEC_FEATURE = feature(
                     flags = [
                         # Disable Exceptions
                         "-lc",
-                        "-lnosys",
+                        "-lstdc++",
+                        "--lnosys",
```

Comment:
Looks like there's an extra hyphen: should be `-lnosys`, not `--lnosys`.

Reply:
Weird I don't even know how I did that